pandas/core/groupby/groupby.py

```diff
@@ -1188,10 +1188,18 @@ def std(self, ddof=1, *args, **kwargs):
         ddof : integer, default 1
             degrees of freedom
         """
-
-        # TODO: implement at Cython level?
         nv.validate_groupby_func('std', args, kwargs)
-        return np.sqrt(self.var(ddof=ddof, **kwargs))
+        if ddof == 1:
+            try:
+                return self._cython_agg_general('std', **kwargs)
+            except Exception:
```

Comment:
this is way too complicated

Reply:
LMK if the update is better.

Reply:
hmm, this is what we are doing for var now...